clustering/marathon.py

```diff
@@ -0,0 +1,555 @@
+
+  docker_parameters:
+    required: false
+    description:
+     - Arbitrary parameters for the Docker container. Ignored if container is defined.
+
+  container_volumes:
+    required: false
+    description:
+     - Array of volumes for the container. Ignored if container is defined.
+
+  env:
+    required: false
+    description:
+     - Key value pairs that get added to the environment variables of the process to start.
+
+  constraints:
+    required: false
+    description:
+     - Valid constraint operators are one of ["UNIQUE", "CLUSTER", "GROUP_BY"].
+
+  acceptedResourceRoles:
+    required: false
+    description:
+     - A list of resource roles.
+
+  labels:
+    required: false
+    description:
+     - Attaching metadata to apps can be useful to expose additional information to other services, so we added the ability to place labels on apps.
+
+  uris:
+    required: false
+    description:
+     - URIs defined here are resolved, before the application gets started. If the application has external dependencies, they should be defined here.
+
+  storeUrls:
+    required: false
+    description:
+     - a sequence of URIs, that get fetched on each instance, that gets started. The artifact could be fetched directly from the source, or put into the artifact store. One simple way to do this is automatic artifact storing.
+
+  dependencies:
+    required: false
+    description:
+     - A list of services upon which this application depends.
+
+  healthChecks:
+    required: false
+    description:
+     - An array of checks to be performed on running tasks to determine if they are operating as expected.
+
+  backoffSeconds:
+    required: false
+    description:
+     - Configures exponential backoff behavior when launching potentially sick apps. The backoff period is multiplied by the factor for each consecutive failure until it reaches maxLaunchDelaySeconds.
+
+  backoffFactor:
+    required: false
+    description:
+     - Configures exponential backoff behavior when launching potentially sick apps. The backoff period is multiplied by the factor for each consecutive failure until it reaches maxLaunchDelaySeconds.
+
+  maxLaunchDelaySeconds:
+    required: false
+    description:
+     - Configures exponential backoff behavior when launching potentially sick apps. The backoff period is multiplied by the factor for each consecutive failure until it reaches maxLaunchDelaySeconds.
+
+  upgradeStrategy_minimumHealthCapacity:
+    required: false
+    description:
+     - a number between 0and 1 that is multiplied with the instance count. This is the minimum number of healthy nodes that do not sacrifice overall application purpose.
+
+  upgradeStrategy_maximumOverCapacity:
+    required: false
+    description:
+     - a number between 0 and 1 which is multiplied with the instance count. This is the maximum number of additional instances launched at any point of time during the upgrade process.
+
+  force:
+    required: false
```

Comment:
if required: false is defined you must define a default e.g. default: null also see http://docs.ansible.com/ansible/developing_modules.html#module-checklist